packages/commonwealth/client/scripts/views/pages/NotificationSettings/CommentSubscriptionEntry.tsx

```diff
@@ -60,9 +50,27 @@ export const CommentSubscriptionEntry = (
   const navigate = useCommonNavigate();
 
   const handleNavigateToThread = () => {
+    if (!thread) {
+      return;
+    }
+
+    const threadUrl = getThreadUrl(
+      {
+        chain: `${thread.community_id}`,
+        id: `${thread.id}`,
+        title: decodeURIComponent(thread.title),
+      },
+      comment_id,
+      true,
+    );
+
     navigate(threadUrl);
   };
 
+  if (!thread) {
```

Comment:
We shouldn't get to this part of the code right? Since we have another if guard at the start of this function

Reply:
Both are needed.  They are there to resolve compilation issues. One is needed inside handleNavigateToThread and the other in the main body. 

I don't like this hack but the only other way to solve this is to go through and find all the broken records in the DB.